src/wiki/getting-started/installing-polymc.md

```diff
@@ -57,6 +57,12 @@ Depending on your system, you may need to grant the PolyMC AppImage **executable
 You can do this by opening your system's terminal application, **making sure to navigate to the location of the downloaded AppImage,** before granting the execute permission using this command:
 
 ```
-sudo chmod +x PolyMC-Linux-1.1.0-x86_64.appimage
+sudo chmod +x PolyMC-Linux-1.2.0-x86_64.appimage
```

Comment:
dont like this, maybe call https://github.com/PolyMC/polymc.github.io/blob/master/src/_data/version.js somehow like download page does
+ suggest appimagelauncher https://github.com/TheAssassin/AppImageLauncher

Reply:
appimagelauncher requires systemd = cringe

Reply:
just suggest it its more user friendly + makes desktop shortcut

Reply:
version thing can we fix it?

Reply:
look now maybe? idk

Reply:
also on the machine where i am i can't compile the website

Reply:
Illegal instruction
error Command failed with exit code 132.

Reply:
bruh sphagetti ima just pr that in another then maybe

Reply:
or wait does it work

Reply:
no idea i literally cannot check